### PR TITLE
Feature/settings screen

### PR DESCRIPTION
**Pull request by NotTrulyEvil**
**Might introduce a bug when trying to reopen the main window from the system tray**

### CODE REVIEW

File: src/main/java/com/github/cypher/gui/root/settings/SettingsPresenter.java [hunk 1]
```diff
@@ -21,9 +22,82 @@
...
 	private Button logoutButton;
 
 	@FXML
+	private Label changesRequireRestartLabel;
+
+	@FXML
+	private void initialize() {
+		switch (settings.getLanguage().getLanguage()) {
+			case "en":
+				languageChoiceBox.setValue("English");
```
Comment (Gurgy) [inline src/main/java/com/github/cypher/gui/root/settings/SettingsPresenter.java]: https://docs.oracle.com/javase/7/docs/api/java/util/Locale.html#getDisplayLanguage(java.util.Locale) might be a better option for higher compatibility

File: src/main/java/com/github/cypher/gui/root/roomcollection/room/chat/chat.fxml [hunk 2]
```diff
@@ -1,14 +1,10 @@
 <?xml version="1.0" encoding="UTF-8"?>
 
-<?import java.lang.String?>
-<?import javafx.scene.control.Button?>
-<?import javafx.scene.control.Label?>
-<?import javafx.scene.control.ListView?>
-<?import javafx.scene.control.TextArea?>
-<?import javafx.scene.layout.AnchorPane?>
+<?import javafx.scene.control.*?>
+<?import javafx.scene.layout.*?>
```
Comment (TheFeffe) [inline src/main/java/com/github/cypher/gui/root/roomcollection/room/chat/chat.fxml]: Unnecessary package imports.

File: src/main/java/com/github/cypher/gui/root/roomcollection/room/room.fxml [hunk 3]
```diff
@@ -4,12 +4,12 @@
 <?import javafx.scene.control.ListView?>
 <?import javafx.scene.layout.*?>
 <?import javafx.scene.text.Font?>
-<StackPane fx:id="mainStackPane" xmlns="http://javafx.com/javafx/8.0.111" xmlns:fx="http://javafx.com/fxml/1" fx:controller="com.github.cypher.gui.root.roomcollection.room.RoomPresenter">
+<StackPane fx:id="mainStackPane" minWidth="0" xmlns="http://javafx.com/javafx/8.0.111" xmlns:fx="http://javafx.com/fxml/1" fx:controller="com.github.cypher.gui.root.roomcollection.room.RoomPresenter">
```
Comment (TheFeffe) [inline src/main/java/com/github/cypher/gui/root/roomcollection/room/room.fxml]: This change might be unnecessary.

File: src/main/java/com/github/cypher/main.css [hunk 4]
```diff
@@ -36,6 +36,7 @@ AnchorPane {
 * {
 	-fx-color-base: #404040;
 	-fx-lighter: #808080;
+	-fx-very-light: #DBDBDB;
```
Comment (TheFeffe) [inline src/main/java/com/github/cypher/main.css]: Should maybe be part of another pr.
